pylint/extensions/magic_value.py

```diff
@@ -74,11 +94,21 @@ def _check_constants_comparison(self, node: nodes.Compare) -> None:
                 confidence=HIGH,
             )
 
-    def _is_magic_value(self, node: nodes.NodeNG) -> bool:
+    def _is_magic_value(self, node: nodes.Const) -> bool:
```

Comment:
`mypy` probably won't like this when `astroid` finally has `py.typed` but that is something for another day.

Reply:
was already Const typing (maybe you compared previous commit?)